doc/continuous_deployment/Devenv_in_local_env.md

```diff
@@ -0,0 +1,57 @@
+# Devenv in local environment
+
+This document describes the steps required to provision a development environment in local VMware environment.
+
+## Testbed setup
+
+1. Clone 2 VMs from `win2016core-bare` template
+    - single ESXi host
+    - destination folder: `Dev\[initials]`
```

Comment:
1. Better would be: `In "Select a name and folder" tab, select Codilime\Dev\[your initials] folder.`

2. Also, this folder must exist first. Folder creation step should be described before the current one.

Reply:
3. Also, we should tell the user how to name their VM. If name is not important, then tell that it's not important and they can choose.

Reply:
4. In "Select a Compute Resource" tab select any ESXi host.
* Remember which one was chosen (will be needed later). The one with the lower IP address is ESXi host 1, the one with higher IP adress is ESXi host 2.